src/Tasks/DownloadFile.cs

```diff
@@ -181,7 +185,11 @@ private async Task DownloadAsync(Uri uri, CancellationToken cancellationToken)
                         {
                             Log.LogMessageFromResources(MessageImportance.High, "DownloadFile.Downloading", SourceUrl, destinationFile.FullName, response.Content.Headers.ContentLength);
 
+#if RUNTIME_TYPE_NETCORE
+                            using (Stream responseStream = await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false))
+#else
                             using (Stream responseStream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false))
+#endif
```

Comment:
Stylistically I'd prefer

```suggestion
                            using (Stream responseStream = await response.Content.ReadAsStreamAsync(
#if RUNTIME_TYPE_NETCORE
                                    // The overload that takes a token is available only on .NET 5+
                                    cancellationToken
#endif
                                ).ConfigureAwait(false))
```